selfdrive/car/subaru/carstate.py

```diff
@@ -135,11 +151,34 @@ def get_common_global_es_signals():
       ("Cruise_Set", "ES_Distance"),
       ("Cruise_Resume", "ES_Distance"),
       ("Signal6", "ES_Distance"),
+
+      ("COUNTER", "ES_Status"),
+      ("Signal1", "ES_Status"),
+      ("Cruise_Fault", "ES_Status"),
+      ("Cruise_RPM", "ES_Status"),
+      ("Signal2", "ES_Status"),
+      ("Cruise_Activated", "ES_Status"),
+      ("Brake_Lights", "ES_Status"),
+      ("Cruise_Hold", "ES_Status"),
+      ("Signal3", "ES_Status"),
+
+      ("COUNTER", "ES_Brake"),
+      ("Signal1", "ES_Brake"),
+      ("Brake_Pressure", "ES_Brake"),
+      ("Signal2", "ES_Brake"),
```

Comment:
`ES_Status.Signal2` no longer exists, and kills controlsd when the checks run.

This is likely `ES_Brake.AEB_Status` now... which already exists on line 132.
```suggestion

```